src/core/reducers/infoDialog.js

```diff
@@ -1,16 +1,52 @@
-import { CLOSE_INFO, SHOW_INFO } from 'core/constants';
+/* @flow */
+export const CLOSE_INFO: 'CLOSE_INFO' = 'CLOSE_INFO';
+export const SHOW_INFO: 'SHOW_INFO' = 'SHOW_INFO';
```

Comment:
why move this out of constants?

also, I've seen this in other places too but isn't the above the same as: `export const SHOW_INFO: 'SHOW_INFO';` ?

Reply:
so I moved them because we want to have ducks everywhere (see #1299).

Reply:
We do this because Flow does not read the value of a constant. For Flow, the type of a constant is always `string`, unless we specify our own type `: 'OUR_TYPE'`. That way, we have a specific type that is not `string` but the constant value.